orchagent/aclorch.cpp

```diff
@@ -3974,6 +4005,19 @@ bool AclOrch::addAclTable(AclTable &newTable)
             m_mirrorV6TableId[table_stage] = table_id;
         }
 
+        // We use SAI_ACL_BIND_POINT_TYPE_SWITCH for PFCWD table in DNX platform.
+        // This bind type requires to bind the table to switch.
+        string platform = getenv("platform") ? getenv("platform") : "";
+        string sub_platform = getenv("sub_platform") ? getenv("sub_platform") : "";
+        if (platform == BRCM_PLATFORM_SUBSTRING && sub_platform == BRCM_DNX_PLATFORM_SUBSTRING &&
+            newTable.type.getName() == TABLE_TYPE_PFCWD)
+        {
+            if(!bindEgrAclTableToSwitch(newTable))
```

Comment:
> Instead of binding ACL table directly to switch, can we use ACL Table Group? If we use ACL Table, we could bind only one ACL to the switch bind point. ACL Table groups are used for port binding.

This was discussed in a previous meeting with both MSFT and BRCM. We understood the limitation, and decided to deferred this change later when we encounter the actual use case, i.e.,  we have multiple egress tables that need to bind to switch. @vmittal-msft

Reply:
Yes. This is was discussed in BRCM meeting and decided to bind acl table instead of group since there no current use case for this. If need arise in future, we can enhance this and support.